.azure-pipelines/azure-pipelines-build.yml

```diff
@@ -186,3 +186,13 @@ jobs:
           fi
         displayName: "Build sonic image"
       - template: .azure-pipelines/check-dirty-version.yml@buildimage
+      - task: ManifestGeneratorTask@0
+        condition: ne(variables.SKIP_MG, 'y')
```

Comment:
We need to add a condition only for Official build, right?

Reply:
We can set SKIP_MG in PR validation. This is convenient for test.